.github/workflows/build.yml

```
@@ -38,19 +38,19 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: [14, 16, 18]
+        node-version: [16, 18]
```

```suggestion
        node-version: [18]
```
Haven't we decided to drop `16` too?